src/plugins/odyssey/instance.js

```diff
@@ -0,0 +1,226 @@
+// @flow
+
+/**
+ * Core "model" logic for the Odyssey plugin.
+ * Basically allows creating a data store of priorities, contributions, and people,
+ * and compiling that data store into a cred Graph.
+ */
+import {
+  Graph,
+  EdgeAddress,
+  NodeAddress,
+  type NodeAddressT,
+  type GraphJSON,
+  sortedNodeAddressesFromJSON,
+} from "../../core/graph";
+
+import deepEqual from "lodash.isequal";
+
+import {
+  NODE_PREFIX,
+  EDGE_PREFIX,
+  type OdysseyNodeTypeIdentifier,
+  isOdysseyNodeTypeIdentifier,
+  type OdysseyEdgeTypeIdentifier,
+  isOdysseyEdgeTypeIdentifier,
+} from "./declaration";
+
+import {toCompat, fromCompat, type Compatible} from "../../util/compat";
+
+import * as NullUtil from "../../util/null";
+
+export type Node = {|
```

Comment:
TODO (not this PR): add weights (except the interpretation of a 'weight' here will probably be as a pagerank seed?)